coordinator/persistence-api/src/main/java/io/stargate/db/metrics/api/ClientInfoMetricsTagProvider.java

```diff
@@ -28,9 +32,35 @@
  */
 public interface ClientInfoMetricsTagProvider {
 
-  /** Returns default interface implementation, which returns empty tags for all methods. */
+  /**
+   * Returns default interface implementation, which returns empty tags or with driver info for all
+   * methods.
+   */
   ClientInfoMetricsTagProvider DEFAULT = new ClientInfoMetricsTagProvider() {};
 
+  /**
+   * Returns tags for a {@link ClientInfo}.
```

Comment:
`Returns tags for a {@link ClientInfo} enriched with the driver tags if they are available`